ui.af.apps/src/main/content/jcr_root/apps/core/fd/af-clientlibs/core-forms-components-runtime-all/resources/i18n/en.json

```diff
@@ -36,5 +36,7 @@
     "years": "years",
     "more": "more",
     "less": "less"
-  }
-}
\ No newline at end of file
+  },
+  "saveDraftSuccessMessage"     :   "The draft is saved successfully.",
```

Comment:
Please add translations in all files

Reply:
I had discussion with @s1mahanty , We add string only in english version and then raise JIRA for translation team, and they translate to other langauges.